lib/cryptoexchange/exchanges/bitstamp/services/pairs.rb

```diff
@@ -0,0 +1,26 @@
+module Cryptoexchange::Exchanges
+  module Bitstamp
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        MARKET = Bitstamp::Market
```

Comment:
Actually I was just thinking why we need to define a constant as another constant, that's about it. 

And actually I think we don't need to hard code markets since we can grab the class name (as you say) and transform it (`underscore` it or something like that)